packaging/appimages/universal-robots-aarch64.yml

```diff
@@ -3,8 +3,7 @@ version: 1
 script:
  - rm -rf AppDir || true
  - mkdir -p $TARGET_APPDIR/usr/bin $TARGET_APPDIR/usr/lib $TARGET_APPDIR/src/
- - cp ../../build/universal-robots $TARGET_APPDIR/usr/bin/
- - cp ../../build/_deps/universal_robots_client_library-build/liburcl.so $TARGET_APPDIR/usr/lib/
+ - cp -r ../../build/install/usr/* $TARGET_APPDIR/usr
```

Comment:
Now we don't need to pick out the individual libraries, or reach into subdirectories of `build` like `_deps` to get our targets: they are all staged into `build/install` for us. So just bulk copy that tree into the `/usr` tree of the app image.